<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnInFile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать вх. файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditInFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRun">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Обработать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditOutFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnOutFile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать вых. файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnOutDir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать вых. директорию</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditInDir">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditOutDir">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnInDir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать вх. директорию</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RadioFile">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вкл</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="RadioDir">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вкл</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="CheckChangeBits">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Переставить биты</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
